constrained-voucher.mkd

```diff
@@ -177,59 +177,45 @@ When the Pledge is known by MASA to support PKIX format certificates, the "pinne
 the Registrar, or the certificate of a domain CA of the Registrar's domain. However, if the Pledge is known to also support RPK pinning and the MASA intends to pin the Registrar's identity (not a CA), then
 MASA MAY pin the RPK of the Registrar instead of the Registrar's End-Entity certificate in order to save space in the voucher.
 
-# Discovery and URI {#discovery-uri}
+When discovering the root path for the EST resources, the server MAY return
+the full resource paths and the used content types. This is useful when
+multiple content types are specified for EST-coaps server. For example, the
+following more complete response is possible.
+
```

Comment:
The above 4 lines appear to be out of place in this section; I would remove the text here.
It may fit somewhere else but without context it is unclear what these lines try to say.

Reply:
Hi, I have pushed another commit to move this text elsewhere, removing the last sentence too.